docs/submission/dictionary-overview.md

```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
 
 - Other fields must meet a regular expression for their value.
 
-## Dictionary Reference Databases
+## Dictionary Standards Used
+
+The dictionary codelist values were derived from external standards or common terminology used by ICGC-ARGO projects. These include: 
+
+- [American Joint Committee on Cancer Staging Classifications](https://cancerstaging.org/references-tools/deskreferences/Pages/Cancer-Staging-Forms.aspx)
+- [World Health Organization International Classification of Diseases, 10th Revision (ICD-10)](https://icd.who.int/browse10/2019/en)
+- [International Classification of Diseases for Oncology (ICD-O))](http://www.iacr.com.fr/index.php?option=com_content&view=category&layout=blog&id=100&Itemid=577)
+- [Cancer Data Standards Registry and Repository (caDSR)](https://cdebrowser.nci.nih.gov/cdebrowserClient/cdeBrowser.html#/search)
+- [Cancer Care Ontario Data Book Reporting Standards](https://www.cancercareontario.ca/en/data-book-reporting-standards)
+- [RxNorm](https://www.nlm.nih.gov/research/umls/rxnorm/index.html)
+- [Common Terminology Criteria for Adverse Events (CTCAE)](https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm)
+- [ECOG-ACRIN Cancer Research Group](https://ecog-acrin.org/resources/ecog-performance-status)
+ 
+
+## Tumour staging classifications
```

Comment:
capitalize "Staging Classifications"

Reply:
I fixed all the casings to make them consistent.